codec-mqtt/src/main/java/io/netty/handler/codec/mqtt/MqttReasonCodes.java

```diff
@@ -0,0 +1,419 @@
+/*
+ * Copyright 2023 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.netty.handler.codec.mqtt;
+
+/**
+ * Provides a set of enumeration that exposes standard MQTT 5 reason codes used by various messages.
+ * */
+public final class MqttReasonCodes {
```

Comment:
A private constructor seems to be needed for this class

Reply:
Fixed by 0be12de9a2aa30debe8a831b85b4302ff4bd768f